content-moderation-evaluation/Evaluating-Amazon-Rekognition-Content-Moderation-Service.ipynb

```diff
@@ -0,0 +1,542 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Notebook for Evaluating Content Moderation Service\n",
+    "\n",
+    "This notebook is part of AWS blog series of Evaluating Content Moderation Service, which provide sample code to streamline steps from creating ground truth labeling job to generating evaluation metrics.\n",
+    "\n",
+    "Prerequisite: **You must prepare image dataset (evaluation dataset) ready for evaluation and [upload them to a S3 bucket](https://docs.aws.amazon.com/redshift/latest/dg/tutorial-loading-data-upload-files.html). The dataset should contain the moderation labels of interest to your use case.** \n",
```

Comment:
I think linking to Redshift docs is a bit confusing. Can we find something better?

Also,
```suggestion
    "Prerequisite: **You must prepare an image dataset (evaluation dataset) ready for evaluation and [upload it to a S3 bucket](https://docs.aws.amazon.com/redshift/latest/dg/tutorial-loading-data-upload-files.html). The dataset should contain the moderation labels of interest to your use case.** \n",
    
```